relate/settings.py

```diff
@@ -27,17 +29,22 @@
     assert _local_settings_file != os.environ["RELATE_LOCAL_TEST_SETTINGS"]
     _local_settings_file = os.environ["RELATE_LOCAL_TEST_SETTINGS"]
 
-local_settings = {
-        "__file__": _local_settings_file,
-        }
+if not os.path.isfile(_local_settings_file):
+    raise RuntimeError(
+        "Management command '%(cmd_name)s' failed to run "
+        "because '%(local_settings_file)s' is missing."
+        % {"cmd_name": sys.argv[1],
+           "local_settings_file": _local_settings_file})
+
+local_settings_module_name = (
+    os.path.splitext(os.path.split(_local_settings_file)[-1])[0])
+exec("import %s as local_settings" % local_settings_module_name)
+
+# Suppress syntax warning in following scripts
 try:
-    with open(_local_settings_file) as inf:
-        local_settings_contents = inf.read()
-except IOError:
+    local_settings = local_settings  # type: ignore  # noqa
```

Comment:
```
local_settings = local_settings_module.__dict__
```
? And then back out all the `getattr` changes below?

Reply:
That's smart.